.github/workflows/main.yml

```diff
@@ -94,21 +94,61 @@ jobs:
         cmake -B build \
           -DBUILD_SHARED_LIBS=ON \
           -DCMAKE_BUILD_TYPE=Release \
+          -DSDL2IMAGE_DEPS_SHARED=OFF \
```

Comment:
This is worth an issue, I think.
We should add the paths of the vendored dependencies + SDL2 to `PATH` on Windows (I think) , to `LD_LIBRARY_PATH` on Linux and `DYLD_LIBRARY_PATH` on Macos.

Reply:
The paths where the libraries get compiled are an implementation detail of the vendored dependencies, and different for each vendored dependency, so that's going to be quite a long search path. Setting the right environment variable is going to require platform-specific knowledge: libtool and CMake both know how to do this for ordinary linking with `-lfoo`, but not for dynamic loading via `dlopen()`.

Reply:
I'm opening an issue.

Reply:
#259